vNext/channels/applicationinsights-channel-js/src/EnvelopeCreator.ts

```diff
@@ -207,9 +207,21 @@ export abstract class EnvelopeCreator {
         //   }
 
         let tgs = {};
-        item.tags.forEach(tg => {
-            tgs = { ...tgs, ...tg };
-        });
+        let tagsCopy = item.tags;
```

Comment:
Is it possible to do this without creating a copy?

Reply:
yes, just removed the copy and make sure all tests passed